assets/src/edit-story/components/checklist/karma/checklist.karma.js

```diff
@@ -52,37 +56,53 @@ describe('Checklist integration', () => {
   describe('open and close', () => {
     it('should toggle the checklist', async () => {
       const { toggleButton } = fixture.editor.checklist;
-      expect(fixture.editor.checklist.issues).toBeNull();
+      expect(
+        fixture.editor.checklist.issues.getAttribute('data-isexpanded')
```

Comment:
Now that this is always rendered we could change [this `queryByRole`](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/karma/fixture/containers/checklist.js#L37-L41) to `getByRole`